### PR TITLE
Feature/webhooks

### PR DESCRIPTION
**Pull request by drobnikj**

### CODE REVIEW

File: src/acts.js [hunk 1]
```diff
@@ -772,4 +772,45 @@ export default {
             .then(pluckData)
             .then(parseDateFields);
     },
+
+    /**
+     * Gets list of act webhooks.
```
Comment (jancurn) [inline src/acts.js]: In new texts we should write "actors". In fact, we should update this in all texts.

File: src/webhook_dispatches.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+     * Gets webhook dispatch.
+     * @memberof ApifyClient.webhookDispatches
+     * @instance
+     * @param {Object} options
+     * @param options.token
+     * @param option.webhookDispatchId - Webhook dispatch ID
+     * @param callback
+     * @returns {WebhookDispatch}
+     */
+    getWebhookDispatch: (requestPromise, options) => {
```
Comment (jancurn) [inline src/webhook_dispatches.js]: You have `listDispatches` function in `webhooks` and `listWebhookDispatches ` and `getWebhookDispatch` here. This should be consistent - I'd call the functions here `listDispatches` and `getDispatch`.

File: src/webhook_dispatches.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        return requestPromise({
+            url: `${baseUrl}${BASE_PATH}`,
+            json: true,
+            method: 'GET',
+            qs: query,
+        })
+            .then(pluckData)
+            .then(parseDateFields);
+    },
+    /**
```
Comment (mtrunkat) [inline src/webhook_dispatches.js]: Just tiny little thing - it's nice to separate methods by empty lines :)

```
},
[EMPTY LINE HERE]
/**
 * ...
```

File: src/webhook_dispatches.js (hunk 2 above)
Comment (drobnikj) [reply src/webhook_dispatches.js]: I followed the API structure:
/webhook-dispatches -> webhookDispatches.listWebhookDispatches()
/webhook-dispatches/[id] -> webhookDispatches.getWebhookDispatch()
/webhooks/[id]/dispatches -> webhooks.listDispatches()

It is true that we can simplify it to 
/webhook-dispatches -> webhookDispatches.listDispatches()
/webhook-dispatches/[id] -> webhookDispatches.getDispatch()

**Comment by drobnikj** [comment]
@jancurn @mtrunkat  can you check the last commit(runAct,Task with ad hoc webhooks) plus changes from PR?

I want to merge it.

File: src/utils.js [hunk 4]
```diff
@@ -274,3 +274,11 @@ export function parseDateFields(obj, depth = 0) {
         return val;
     });
 }
+
+/**
+ * Helper function that converts array of webhooks to base64 string
+ */
+export function stringifyWebhoohsToBase64(webhooks) {
```
Comment (mnmkng) [inline src/utils.js]: Should be `...Webhooks...` not `Webhoohs`

File: src/utils.js [hunk 5]
```diff
@@ -274,3 +274,11 @@ export function parseDateFields(obj, depth = 0) {
...
+    const webhhoksJson = JSON.stringify(webhooks);
```
Comment (mnmkng) [inline src/utils.js]: `webhkoks` -> `webhooks`

**Comment by mtrunkat** [comment]
Looks good, just test for `stringifyWebhoohsToBase64()` is missing :)

**Pull request merged**